src/shared/telemetries/telemetriesconfig/config.go

```diff
@@ -7,6 +7,7 @@ import (
 
 const (
 	TelemetryAPIAddressKey         = "telemetry-address"
+	TelemetryAPIAddressDefault     = "http://localhost:8081/api/telemetry/query"
```

Comment:
Doesn't this make deployments without helm charts disabled by default? Is there a way to config that only if run from the IDE or something

Reply:
Fixed the config, thanks. The address no longer changes client-side, instead the server handles determining the release stage based on the version.